package.json

```diff
@@ -77,6 +77,7 @@
     "msw": "^2.0.7",
     "netlify-cli": "^20.1.1",
     "next": "^15.0.0-canary.28",
+    "next-next": "npm:next@^15.3.1-canary.7",
```

Comment:
This is to get types for new `use cache` handler. I didn't want to touch `next` version we already use to not increase scope of this PR, as there is a lot of `next` types issues to solve in general.

Maybe this alias should be named better to indicate it's for `use cache` specifically (at least as of now)